config/application.rb

```diff
@@ -1,6 +1,12 @@
 require_relative 'boot'
 
 require 'rails/all'
```

Comment:
Would you need to remove anything here?

Reply:
Haha, yep, need to remove that line!

I've done this now, and made some more changes to make this actually pass the tests. Thanks for taking a look Pablo :)